packages/office-ui-fabric-react/src/components/Facepile/Facepile.base.tsx

```diff
@@ -90,7 +90,7 @@ export class FacepileBase extends BaseComponent<IFacepileProps, {}> {
         ? onRenderPersona(persona, this._getPersonaControl)
         : onRenderPersonaCoin(persona, this._getPersonaCoinControl);
       return (
-        <li role="option" key={`${singlePersona ? 'persona' : 'personaCoin'}-${index}`} className={this._classNames.member}>
+        <li role="presentation" key={`${singlePersona ? 'persona' : 'personaCoin'}-${index}`} className={this._classNames.member}>
```

Comment:
The snapshots need to be updated. You can start the tests in watch mode using `npm run start-test` (office-ui-fabric-react pacakge) and press `u` to update the snapshots.

You might need to update some other snapshots too for examples. But i recommend you to run a full `npm run build` in the root, so you do not need to wait for CI each time :D